test/compflags/kushal/llvm-incremental.bad

```diff
@@ -0,0 +1 @@
+Incremental compilation is not yet supported with LLVM
```

Comment:
I believe this is not going to match the output generated.  USR_FATAL prefixes these messages with the file name, line number, and "error:".  The .bad output should include what the compiler generates at this point (so verify by running the future with everything set up right)